src/Components/Aspire.Azure.Storage.Blobs/AspireBlobStorageExtensions.BlobStorageContainerComponent.cs

```diff
@@ -55,7 +57,7 @@ protected override void BindSettingsToConfiguration(AzureBlobStorageContainerSet
         }
 
         protected override IHealthCheck CreateHealthCheck(BlobContainerClient client, AzureBlobStorageContainerSettings settings)
-            => new AzureBlobStorageContainerHealthCheck(client);
+            => new AzureBlobStorageHealthCheck(client.GetParentBlobServiceClient(), new AzureBlobStorageHealthCheckOptions { ContainerName = client.Name });
```

Comment:
Don't we want to check that the container exists?

Reply:
That's what the `AzureBlobStorageHealthCheckOptions` is here for